plugins/se.infomaker.newspilot.job/JobPackage.js

```diff
@@ -72,6 +72,11 @@ export default {
             sv: 'Loggar in, vänligen vänta'
         })
 
+        config.addLabel('unsupported-format', {
+            en: 'Unsupported image format in the Writer',
```

Comment:
"Unsupported image format in the Writer" blir eventuellt lite knepig engelska? Räcker nog med "Unsupported image format"